ibis/sql/alchemy.py

```diff
@@ -894,3 +911,61 @@ def _floor_divide(t, expr):
         return t.translate(new_expr)
 
     return fixed_arity(lambda x, y: x / y, 2)(t, expr)
+
+
+@compiles(ops.SortKey)
+def _sort_key(t, expr):
+    # We need to define this for window functions that have an order by
+    by, ascending = expr.op().args
+    sort_direction = sa.asc if ascending else sa.desc
+    return sort_direction(t.translate(by))
+
+
+_valid_frame_types = numbers.Integral, str, type(None)
+
+
+class Over(_Over):
```

Comment:
I have a PR to SQLAlchemy to add frame specification to `Over` elements: https://github.com/zzzeek/sqlalchemy/pull/264. Just an FYI, and we don't need to wait for that PR to merge this one.